Pass truncate argument to pg:locks

Catalectic opened the pull request:
Having untrucated query strings in other pg methods is very handy, would it be possible to have the same option for pg:locks, eg something like this?

jeffday commented:
:+1: this is helpful when troubleshooting lock contention in a production environment - otherwise i have to look at the PID and just go into `pg_stat_activity` anyway to figure out what's going on

will commented:
Can one of you do me a favor and now that there'd be 3 cases of the truncate, refactor it out into a private method?

https://github.com/heroku/heroku-pg-extras/blob/master/lib/heroku/command/pg.rb#L574
and 
https://github.com/heroku/heroku-pg-extras/blob/master/lib/heroku/command/pg.rb#L574

Catalectic commented:
I've added a commit to avoid that duplication, handling pg:calls/pg:outliers and pg:locks was a little awkward, let me know if you have any suggestions on how to make this clearer.

I've updated my master branch so this can be tested: heroku plugins:install git://github.com/Catalectic/heroku-pg-extras.git

will commented:
I changed it up a little bit in cafe25450e5c7a69c7670ea1281415c181195d84 but overall good.

Thanks!
